modules/web-console-configuration.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * web_console/configuring-web-console.adoc
+
+[id="web-console-configuration_{context}"]
+= Configuring the web console
+
+You can configure image registry settings by editing the
+`image.config.openshift.io/cluster` resource.
+
+* Edit the `consoles.operator.openshift.io resource:
++
+----
+$ oc edit consoles.operator.openshift.io
+----
++
+The following example displays the parameters from this resource that you can
+modify:
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Console
+metadata:
+  name: cluster
+spec:
+  managementState: Managed <1>
+  authentication:
+    logoutRedirect: "" <2>
+status:
+  consoleURL: "" <3>
+----
+<1> Specify the state of the console, `Managed` or `Disabled`. If the value is
+`Managed`, then the web console is enabled. Set it to `Disabled` to disable
+the console.
+<2> Specify the URL of the page to load when a user logs out of the web console.
```

Comment:
@benjaminapetersen, is this right?
@huffmanca, have you heard about this SSO requirement?